File: examples/swagger-2/src/test/resources/asserts/java/spring_cloud_openfeign/src/com/wuxp/codegen/swagger2/model/paging/Page.java
package com.wuxp.codegen.swagger2.model.paging;

import lombok.Data;
import lombok.experimental.Accessors;
import javax.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import com.wuxp.codegen.swagger2.model.paging.Pageable;
        import com.wuxp.codegen.swagger2.model.paging.Sort;

@Data
@Accessors(chain = true)
public class  Page<T> {

        private List<T> content;

        private String empty;

        private String first;

        private String last;

        private Integer number;

        private Integer numberOfElements;

        private Integer size;

        private Integer totalElements;

        private Integer totalPages;

        private Pageable pageable;

        private Sort sort;

}

File: examples/swagger-2/src/test/resources/asserts/typescript/axios/src/model/paging/Page.ts
/* tslint:disable */
/* eslint-disable */

        import {Pageable} from "./Pageable";
        import {Sort} from "./Sort";
import {DefaultOrderField} from "feign-client";



export interface  Page<T> {

        content: Array<T>;
        empty: boolean;
        first: boolean;
        last: boolean;
        number: number;
        numberOfElements: number;
        size: number;
        totalElements: number;
        totalPages: number;
        pageable: Pageable;
        sort: Sort;
}